diff --git a/src/win/86Box.rc b/src/win/86Box.rc
index 845b6584..3ed35366 100644
--- a/src/win/86Box.rc
+++ b/src/win/86Box.rc
@@ -993,31 +993,11 @@ BEGIN
     IDS_2126	"An emulator of old computers\n\nAuthors: Sarah Walker, Miran Grca, Fred N. van Kempen (waltje), SA1988, MoochMcGee, reenigne, leilei, JohnElliott, greatpsycho, and others.\n\nReleased under the GNU General Public License version 2. See LICENSE for more information."
     IDS_2127	"OK"
     IDS_2128	"Hardware not available"
-#ifdef _WIN32
-#define LIB_NAME_PCAP "WinPcap"
-#else
-#define LIB_NAME_PCAP "libpcap"
-#endif
-    IDS_2129	"Make sure " LIB_NAME_PCAP " is installed and that you are on a " LIB_NAME_PCAP "-compatible network connection."
+    IDS_2129	"Make sure WinPcap is installed and that you are on a WinPcap-compatible network connection."
     IDS_2130	"Invalid configuration"
-#ifdef _WIN32
-#define LIB_NAME_FREETYPE "freetype.dll"
-#else
-#define LIB_NAME_FREETYPE "libfreetype"
-#endif
-    IDS_2131	LIB_NAME_FREETYPE " is required for ESC/P printer emulation."
-#ifdef _WIN32
-#define LIB_NAME_GS "gsdll32.dll"
-#else
-#define LIB_NAME_GS "libgs"
-#endif
-    IDS_2132	LIB_NAME_GS " is required for automatic conversion of PostScript files to PDF.\n\nAny documents sent to the generic PostScript printer will be saved as PostScript (.ps) files."
-#ifdef _WIN32
-#define LIB_NAME_FLUIDSYNTH "libfluidsynth.dll"
-#else
-#define LIB_NAME_FLUIDSYNTH "libfluidsynth"
-#endif
-    IDS_2133	LIB_NAME_FLUIDSYNTH " is required for FluidSynth MIDI output."
+    IDS_2131	"freetype.dll is required for ESC/P printer emulation."
+    IDS_2132	"gsdll32.dll is required for automatic conversion of PostScript files to PDF.\n\nAny documents sent to the generic PostScript printer will be saved as PostScript (.ps) files."
+    IDS_2133	"libfluidsynth.dll is required for FluidSynth MIDI output."
     IDS_2134	"Entering fullscreen mode"
     IDS_2135	"Don't show this message again"
     IDS_2136	"Don't Exit"
