diff --git a/src/codegen/codegen.h b/src/codegen/codegen.h
index d3898b53..c9f33068 100644
--- a/src/codegen/codegen.h
+++ b/src/codegen/codegen.h
@@ -40,6 +40,7 @@
 #include <86box/mem.h>
 #include "x86_ops.h"
 
+#ifdef USE_DYNAREC
 #ifdef __amd64__
 #include "codegen_x86-64.h"
 #elif defined i386 || defined __i386 || defined __i386__ || defined _X86_ || defined _M_IX86 || defined _M_X64
@@ -47,6 +48,7 @@
 #else
 #error Dynamic recompiler not implemented on your platform
 #endif
+#endif
 
 /*Handling self-modifying code (of which there is a lot on x86) :
 
@@ -342,6 +344,7 @@ extern int block_pos;
 
 #define CPU_BLOCK_END() cpu_block_end = 1
 
+#ifdef USE_DYNAREC
 static inline void addbyte(uint8_t val)
 {
         codeblock[block_current].data[block_pos++] = val;
@@ -402,5 +405,5 @@ extern int codegen_fpu_loaded_iq[8];
 extern int codegen_reg_loaded[8];
 
 extern int codegen_in_recompile;
-
+#endif
 #endif
