diff --git a/Alc/helpers.c b/Alc/helpers.c
index d2cb625..32e8cce 100644
--- a/Alc/helpers.c
+++ b/Alc/helpers.c
@@ -218,6 +218,10 @@ void FillCPUCaps(int capfilter)
 #endif
 #endif
 #ifdef HAVE_NEON
+#ifdef WIN32
+#warning "Assuming NEON run-time support!"
+    caps |= CPU_CAP_NEON;
+#else
     FILE *file = fopen("/proc/cpuinfo", "rt");
     if(!file)
         ERR("Failed to open /proc/cpuinfo, cannot check for NEON support\n");
@@ -262,6 +266,7 @@ void FillCPUCaps(int capfilter)
 
         alstr_reset(&features);
     }
+#endif
 #endif
 
     TRACE("Extensions:%s%s%s%s%s%s\n",
