From 899582b3ca7daa6e0fd191e66054d275ee825d6b Mon Sep 17 00:00:00 2001
From: GH Cao <driver1998@foxmail.com>
Date: Fri, 17 Jan 2020 15:21:55 +0800
Subject: [PATCH 2/7] Workaround for undefined NULL on clang/MinGW

---
 src/cpu_new/386_common.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/cpu_new/386_common.h b/src/cpu_new/386_common.h
index 3ac390f9..0453c25f 100644
--- a/src/cpu_new/386_common.h
+++ b/src/cpu_new/386_common.h
@@ -19,6 +19,10 @@
 #ifndef _386_COMMON_H_
 #define _386_COMMON_H_
 
+#ifndef NULL
+#define NULL ((void*)0)
+#endif
+
 #define readmemb(s,a) ((readlookup2[(uint32_t)((s)+(a))>>12]==-1)?readmembl((s)+(a)): *(uint8_t *)(readlookup2[(uint32_t)((s)+(a))>>12] + (uint32_t)((s) + (a))) )
 #define readmemw(s,a) ((readlookup2[(uint32_t)((s)+(a))>>12]==-1 || (((s)+(a)) & 1))?readmemwl((s)+(a)):*(uint16_t *)(readlookup2[(uint32_t)((s)+(a))>>12]+(uint32_t)((s)+(a))))
 #define readmeml(s,a) ((readlookup2[(uint32_t)((s)+(a))>>12]==-1 || (((s)+(a)) & 3))?readmemll((s)+(a)):*(uint32_t *)(readlookup2[(uint32_t)((s)+(a))>>12]+(uint32_t)((s)+(a))))
-- 
2.17.1

